app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,6 @@
     package="com.mapbox.maps.testapp">
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Do we actually need this one btw?

Reply:
can't remember what the use-case was but yes we do :)